cypress/support/commands.ts

```diff
@@ -54,18 +54,31 @@ Cypress.Commands.overwrite(
           onBeforeLoad(win) {
             options?.onBeforeLoad?.(win)
 
-            // We want to test from a clean state, so we clear the persisted state.
             win.indexedDB.deleteDatabase('redux')
 
-            // Since the IndexedDB state is clear, the first migration will run and pick up this legacy state from localStorage.
-            win.localStorage.setItem(
-              'redux_localstorage_simple_user', // storage key for the user reducer using 'redux-localstorage-simple'
-              JSON.stringify({
-                hideUniswapWalletBanner: true,
-                ...CONNECTED_WALLET_USER_STATE,
-                ...(options?.userState ?? {}),
-              })
-            )
+            const dbRequest = win.indexedDB.open('redux')
+
+            dbRequest.onsuccess = function () {
+              const db = dbRequest.result
+              const transaction = db.transaction('keyvaluepairs', 'readwrite')
+              const store = transaction.objectStore('keyvaluepairs')
+              store.put(
+                {
+                  user: {
+                    ...initialState,
+                    hideUniswapWalletBanner: true,
+                    ...CONNECTED_WALLET_USER_STATE,
+                    ...(options?.userState ?? {}),
+                  },
+                },
+                'persist:interface'
+              )
+            }
+
+            dbRequest.onupgradeneeded = function () {
+              const db = dbRequest.result
+              db.createObjectStore('keyvaluepairs')
+            }
```

Comment:
Add a comment explaining what this does. Consider moving it to a helper function, something like

    resetIndexedDB(state: unknown): Promise<void>

I'm not sure if you need to wait for it to be set or not.